lang/gtext_fre.po

```diff
@@ -6809,82 +6810,82 @@ msgstr "Niveaux Perdus"
 #: guitext:976 guitext:977 guitext:978 guitext:979 guitext:980
 msgctxt "Unused"
 msgid "Moo3"
-msgstr ""
+msgstr "Moo3"
```

Comment:
No need to translate this. It's a placeholder and if left empty it will take the string from the original file.

Reply:
alright
Also some traps are not optimally translated but i need more context.